prisma/schema.prisma

```diff
@@ -0,0 +1,100 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id          Int        @id @default(autoincrement())
+  username    String     @unique
+  email       String     @unique
+  first_name  String
+  last_name   String
+  task        Task[]
+  activity    Activity[]
+  role_id     Role?
+  projects    Projects[]
+}
+
+model Projects {
```

Comment:
FYI: The convention for naming models is to use singular nouns instead of plural.

Somewhat of a nitpick, but it does reduce confusion when we are creating a single object (`new Project()`) versus an array of objects (`Project[]`) instead of `const project = new Projects()`.